ui-tests/src/test/resources/features/integrations/ftp-to-ftp.feature

```diff
@@ -25,25 +25,38 @@ Feature: Integration - FTP to FTP
       # select salesforce connection as 'from' point
     When Camilla selects the "FTP" connection
     And she selects "download" integration action
-    And she fills ftp download form with values
-      | fileName | directoryName | initialDelay | delay | delete |
-      | test.txt  | download             | 1000         | 500   | Yes    |
+    And she fills in values
+      | File name expression               | test.xml |
```

Comment:
There should be `test.txt` the `*.xml` suffix is incorrect and test will fail.

Reply:
thanks, mistake